oak/server/asylo/BUILD

```diff
@@ -93,10 +93,20 @@ sgx.debug_enclave(
     config = ":grpc_enclave_config",
 )
 
+# TODO: Use genrule from asylo, when it will become available
+genrule(
+    name = "oak_mrenclave",
+    srcs = [":oak_enclave_signing_material"],
+    outs = ["oak_mrenclave.txt"],
+    # Gets the enclave hash from the enclave signing material as a hex string
+    # 188 is an offset of the hash, 32 is the size of the hash
```

Comment:
Could you add some indication on where these offsets are coming from? Does not have to show the exact computation, but roughly some sources to reconstruct it if necessary.

Reply:
Done